queries.sql

```diff
@@ -8,4 +8,40 @@ select date_of_birth from animals where name = 'Agumon' OR name = 'Pikachu';
 select name, escape_attempts from animals where weight_kg > 10.5;
 select * from animals where neutered = true;
 select * from animals where name != 'Gabumon';
-select * from animals where weight_kg BETWEEN 10.4 AND 17.3;
\ No newline at end of file
+select * from animals where weight_kg BETWEEN 10.4 AND 17.3;
+
+BEGIN;
+UPDATE animals SET species = 'unspecified';
+SELECT name, species FROM animals;
```

Comment:
- [ ] According to the requirements, kindly rollback the changes and verify that the species column went back to its initial state before the transaction. It should look like this:
```BEGIN;
   UPDATE animals SET species = 'unspecified'; 
  SELECT species from animals; (verify that change was made)
   ROLLBACK;
  SELECT species from animals; (verify that change was undone)
```

Reply:
This issue is well resolved! ✔️